docs/api/api.rst

```diff
@@ -1,18 +1,18 @@
 .. _api:
```

Comment:
Would it be useful to rename this whole file `app.rst`?

Reply:
@nZac Yeah, I think yes, will do it.